marklogic-data-hub-central/ui/src/pages/TilesView.test.tsx

```diff
@@ -136,7 +138,7 @@ describe('TilesView component', () => {
         expect(getByTestId('runStep-1')).toBeInTheDocument();
     });
 
-    test('Verify run tile cannot edit with only readStep authority', async () => {
+    test('Verify run tile cannot edit or run with only readStep authority', async () => {
```

Comment:
Can we add a custom step to run.config and verify it shows up in Run tile for these authorities?

Reply:
sure

Reply:
added a custom step and a mapping step